src/DataGroup/Element/CombinedAddress.php

```diff
@@ -28,28 +40,47 @@ class CombinedAddress extends AbstractAddress
      */
     private $addressLine2;
 
-    public function getAddressLine1(): ?string
+    /**
+     * Country (ISO 3166-1 alpha-2)
+     *
+     * @var string
+     */
+    private $country;
+
+    public static function create(
+        string $name,
+        ?string $addressLine1,
+        string $addressLine2,
```

Comment:
First line nullable but second is required?

Reply:
Yes, according to specs.
The first line is used for the street name and number, the second line for the city. In small villages, not every address has a street name, therefore it's optional.